src/server/config.js

```diff
@@ -6,71 +6,30 @@ var fs = require('fs'),
 var config = {};
 var simulationFilePath;
 
+// module properties
+[
+    'liveReload',
+    'platform',
+    'platformRoot',
+    'forcePrepare',
+    'projectRoot',
+    'server',
+    'telemetry',
+    'simHostOptions',
+    'xhrProxy',
+    'touchEvents'
+].forEach(function (prop) {
```

Comment:
This is a nice approach, but we are losing our [optional](https://github.com/Microsoft/cordova-simulate/pull/20/files#diff-852bb59fffeac067fe0e2cbf7646bc73R54) flag here. One solution is to remove the [throw call](https://github.com/Microsoft/cordova-simulate/pull/20/files#diff-852bb59fffeac067fe0e2cbf7646bc73L105) as well to avoid errors when querying for an optional config that was not set.